### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

**Comment by chrfritsch** [comment]
I also opened https://github.com/thunder/testsite_builder/issues/29

File: README.md [hunk 1]
```diff
@@ -1,29 +1,66 @@
...
+    composer require thunder/testsite_builder
+
+Then enable the testsite_builder module, either with command line or in the admin UI.
+
+    drush en testsite_builder
+
+Flush the caches, and you are ready to go.
+
+### Adjust database
+Adjust the MySQL database settings, to use the "--create-content" option.
```
Comment (mtodor) [inline README.md]: I think this should be formulated differently. It was confusing to me. It sounds to me, that I have to change database settings in order to use the `--create-content` option for the database server.

Maybe we can say something like: "..., in order to use content create functionality of the testsite_builder." .. or something like that.

File: src/ContentCreator.php [hunk 2]
```diff
@@ -8,16 +8,6 @@
...
  *
- * *** IMPORTANT ***
- * - we need GRANT:
- *    GRANT FILE on *.* to 'drupaluser'@'localhost'
- * - we need disabled "secure-file-priv":
- *    [mysqld]
- *    secure-file-priv=""
- *
- * TODO:
- *   - add logging (fastest solution is "error_log")
```
Comment (mtodor) [inline src/ContentCreator.php:19]: This `TODO` should stay. 😉

Comment (chrfritsch) [reply src/ContentCreator.php:19]: Created #29 for that

**Review by mtodor — APPROVED** [review]

**Pull request merged**